### ISSUE TITLE
Import error on KDE with latest source

### ISSUE THREAD

**Issue by jat255** [jaraco/keyring#187]
I'm not sure when this happened (I think it might have been a change on the KDE side), but I can no longer import the keyring module. My system is running Arch with KDE Plasma (KWallet 5.17.0), and raises an AttributeError when importing the module, as shown below. Any ideas?

``` python
Python 2.7.11 (default, Dec  6 2015, 15:43:46) 
Type "copyright", "credits" or "license" for more information.

IPython 4.0.0 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: import keyring
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-ef6fc8fd899e> in <module>()
----> 1 import keyring

/home/josh/git_repos/keyring/keyring/__init__.py in <module>()
     10 logger = logging.getLogger('keyring')
     11 
---> 12 from .core import (set_keyring, get_keyring, set_password, get_password,
     13                   delete_password)
     14 from .getpassbackend import get_password as get_pass_get_password

/home/josh/git_repos/keyring/keyring/core.py in <module>()
    171 
    172 # init the _keyring_backend
--> 173 init_backend()

/home/josh/git_repos/keyring/keyring/core.py in init_backend()
     57     """
     58     _load_library_extensions()
---> 59     set_keyring(load_config() or _get_best_keyring())
     60 
     61 

/home/josh/git_repos/keyring/keyring/core.py in load_config()
    126         return
    127 
--> 128     return load_keyring(keyring_name)
    129 
    130 def _load_keyring_path(config):

/home/josh/git_repos/keyring/keyring/core.py in load_keyring(keyring_name)
     94     __import__(module_name)
     95     module = sys.modules[module_name]
---> 96     class_ = getattr(module, class_name)
     97     # invoke the priority to ensure it is viable, or raise a RuntimeError
     98     class_.priority

AttributeError: 'module' object has no attribute 'Keyring'

```

**Comment by mitya57** [comment]
Can it be that you have specified a keyring name that doesn't exist in your config?

**Comment by jaraco** [comment]
Oh, damn. When I renamed kwallet.QtKeyring, I broke those who were specifically referencing kwallet.Keyring.

**Comment by jaraco** [comment]
Released as 7.0.2. Sorry for the trouble.

Do please note the changelog that the 'Keyring' name is now deprecated and you should choose one of QtKeyring or DBusKeyring (with the latter being preferred).

**Comment by jat255** [comment]
Oh, awesome. I figured I had done something wrong. I forget why I had explicitly set the keyring before in my config file, but that appeared to be the issue.  6e36aa7 seems to have fixed it, but I switched my config to `DBusKeyring` anyway. Thanks for the prompt resolution!
